packages/manager-components/src/components/ManagerButton/ManagerButton.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { PropsWithChildren, RefAttributes, HTMLAttributes } from 'react';
+import {
+  OsdsButton,
+  OsdsTooltip,
+  OsdsTooltipContent,
+} from '@ovhcloud/ods-components/react';
+import { JSX } from '@ovhcloud/ods-components';
+import { useTranslation } from 'react-i18next';
+import { StyleReactProps } from '@ovhcloud/ods-components/react/dist/types/react-component-lib/interfaces';
+import './translations';
+
+import { useAuthorizationIam } from '../../hooks/iam';
+
+export type ManagerButtonProps = PropsWithChildren<{
+  iamActions?: string[];
+  urn?: string;
+  isMenuItem?: boolean;
```

Comment:
**suggestion:** We can rename it to `displayTooltip`.
```suggestion
  isMenuItem?: boolean;
```

Reply:
👍